addons/common/functions/fnc_blurScreen.sqf

```diff
@@ -10,6 +10,8 @@
 
 #include "script_component.hpp"
 
+if !(hasInterface) exitWith {};
```

Comment:
I agree with @bux578 on this one. It makes absolutely no sense to have it outside.